### ISSUE TITLE
TWE: Create Template Issue: Research Plan

### ISSUE THREAD

**Issue by bradyse** [hackforla/internship#313]
### Overview
We need to create a template issue for Research Plan so that we have consistent, transparent, defined and self-supporting structure for people creating them.

### Action Items
- [x] Add the milestone: Project Management
- [x] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.
- [x] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [x] If no dependencies, delete dependency section
- [x] Any place you see `[replace...]` replace both the text and the brackets with what it is asking for
- [x] Create a draft of new template by copying the content block below to a comment on this issue, which will be known as our Draft
- [x] In your draft, Fill out the top part (called front matter) with the info it needs.  See the Resources section below for definition and example.
- [x] Define any Dependencies (or remove the dependency section)
- [x] Customize the Overview section
- [x] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [x] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [x] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [x] the links provided by the issue creator should be in the same order as the Action Items 
- [x] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking the WIKI Templates by role pages.  
  - [x] if there is a template
     - [x] Add a link to the template into the ` ### Resources`  section of your draft
  - [x] if there is not template yet
     - [x] Go to [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link under the ` ### Resources`  section (in this top comment) for yourself so that you can create a template later
    - [x] then put a placeholder into the` ### Resources`  section of the draft `[TWE: Template: Research Plan]`
- [x] Get sign-off by product on the draft
- [x] Create the template issue on the repo
- [x] Update the appropriate place in wiki for your template (see resources section)

### Resources/Instructions
[Draft of Google Doc of Research Plan Template](https://docs.google.com/document/d/1SDW6n086PnSrrVSmPgQZWhlkz5_yC70vDvp7Lx7u0Wo/edit?usp=sharing)
[Draft of TWE: How to Write a Research Plan](https://docs.google.com/document/d/1Cwc0w4ZPUI8989w3jU8BW2LzLK_Tl5gHdI0VxN5ej0o/edit?usp=sharing)

#### WIKI Templates
- by role pages
  - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
  - [Design]
  - [Product]
  - [Engineering]
- default document templates, in case you don't have one yet can found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Content to copy for draft
```
Draft of Research Plan template issue
---
name: TWE: Create Template Issue: Research Plan
about: 
title: TWE: [action verb] [name of template]
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources
```
### Resources
<details>
<summary>Front Matter Fields and Meaning</summary>

- name: (name that appear on the new issue tab)
- about: (is the description that appears next to the issue template on the new issue tab
- title: (the title that appears when the issue is created.  User then customizes this title for their issue)
- labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
- assignees: 
</details>

<details>
<summary>Front Matter Field example</summary>

- name: 'Research: Roadmap'
- about: The start of a new roadmap
- title: 'TWE: Research Roadmap [name of type]'
- labels: 'feature: research, feature: roadmap, milestone: missing, role: UI/UX research,
-   size: 0.50pt'
- assignees: ''
</details>

**Comment by bradyse** [comment]
Draft of Research Plan template issue
---
name: TWE: Create Template Issue: Research Plan
about: 
title: TWE: [action verb] [name of template]
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources

**Comment by bradyse** [comment]
## 2022.09.03 Meeting Agenda

### Prework to prep for meeting
- [ ] Review draft of [Draft of Google Doc of Research Plan Template](https://docs.google.com/document/d/1SDW6n086PnSrrVSmPgQZWhlkz5_yC70vDvp7Lx7u0Wo/edit?usp=sharing)
- [ ] Review action items in the [Template Draft ](https://github.com/hackforla/internship/issues/313#issuecomment-1235791661)comment of #313

### Agenda Items
- [ ] Complete remaining steps of the top comment of #313

### Questions
- Front matter: "name" vs. "title" is confusing. It should be "Issue Template Name" and title should be "Issue Title"
- Why does the "Instructions for creating this issue" need to be the in the Draft Comment? Where does the person creating the template issue visit this section? Before or after the filling out of Action Items?
- Once RP is approved, then the recruitment and all that needs to happen. 
     - The final script and questions need to be approved before creating a forward-facing RP.
     - When to create that forward-facing RP?

### Notes from Meeting

### Task Items

**Comment by Nat-nn** [comment]
My update 09/15/22 :
1. Progress:   Updated who benefits  and added links; Added several new terms. 
2. Blockers:   Who benefits - need some clarification; Where located: Are we trying to find only 1 location or want to add all locations where we have it coming up? 
3. Availability: 2-3 hours
4. ETA:  2 weeks (by 09/22/22) 
5. Pictures (if necessary): N/A

**Comment by bradyse** [comment]
1. Progress: Updated Research Plan Template and created a guide for how to write a research plan (link in resources)
2. Blockers: Need Bonnie to review
3. Availability: 1-3 hours per week
4. ETA: After next UX PM meeting
5. Pictures or links (if necessary):

_* remember to add links to the top of the issue if they are going to be needed again._

**Comment by ExperimentsInHonesty** [comment]
@bradyse didn't I already review this.  And I thought the plan was to have Linda create an issue from it.  If I am confusing it with something else, please let me know.

**Comment by bradyse** [comment]
@ExperimentsInHonesty Yes, you did review this, and we then assigned the other issue to Linda. The next item in the action item is product sign-off. If you want to sign-off on this issue template, then please do so, since the template issue is already updated in the issue template repo.

**Comment by ExperimentsInHonesty** [comment]
Draft of Research Plan template issue

---
name: Research: Research Plan
about: The start of a new research plan
title: TWE: Create Research Plan for RP[###]
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: 1pt', 'role: UI/UX research', 'feature: research', 'feature: research plan', 'milestone: missing'
assignees: 
---
### Dependencies
- [ ] [Replace this with Roadmap issue #]
- [ ] A research setup structure (wiki, Github labels, Google Drive, etc.) has been created and approved.

### Overview
We need to create a Research Plan for [NAME OF RESEARCH PLAN] so that [RESEARCH GOAL(S)].

### Action Items
- [ ] Update this issue to make sure it's properly categorized and easy to manage
     - [ ] Under "Projects," add to the Project Management Board (helps with Project Management overview)
     - [ ] Add the milestone: [Research Plan Creation](https://github.com/hackforla/internship/milestone/29) (helps with prioritization)
     - [ ] Add label of the correct research plan number associated with this research plan (e.g., Research: RP001) (helps you find related issues in the same workflow)
     - [ ] Edit title of issue with the correct research plan number (same as prior)
     - [ ] Edit placeholders in Overview of this issue with the relevant information in square brackets. (personalizes the issue to make it easier to read). Any place you see [replace...] replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] Update this issue with the relevant Resources:
     - [ ] Go to the Research Output Overview Page (link in Resources)
       - [ ] find the relevant overview type wiki page. 
       - [ ] change `[Wiki [TYPE] Research Overview Page]` in the Resources below with the correct wiki page of the research overview by type.
     - [ ] Go to the Research Output Overview Page again and find the relevant wiki page for this specific research plan. Then, change the `[RP__ Wiki Page]` in the Resources below with this RP's specific wiki page.
     - [ ] Find and link the folder you are going to put your research plan in  (e.g., Research by Type>2. Intern>RP6 - Intern Intake Interviews)
        - [ ] Go to the Google Drive's Research by Type Folder (link in Resources)
        - [ ] Go into the relevant Type Folder (e.g., 1. Mentor, 2. Intern, etc.) 
        - [ ] once inside, right-click on the relevant destination folder and copy the link. (e.g., RP6 - Intern Intake Interviews)
         - [ ] Add the Google Drive link in the Resources below with the link you just copied and revise the display text to include the Research Plan number (e.g., `Google Drive RP__ Folder` to `Google Drive RP008 Folder`)
- [ ] Create the Research Plan Document.
     - [ ] Make a new document using the TWE Research Plan template (link in Resources). Follow the instructions in the How To Write a Research Plan guide (link in Resources) and mark your progress in this issue.
     - [ ] Move your document from "My Drive" to the shared drive internship folder of this specific RP.
     - [ ] Name your document to be called "TWE: RP___ : [TITLE OF RESEARCH] Research Plan".
     - [ ] Update the document with the relevant information in accordance with the research roadmap document (found on the Research Overview Page).
     - [ ] Update the Table of contents (page 2) when you are finished with the document.
- [ ] Meet and sign-off research plan document with UX Research lead (by adding it to the research team meeting agenda).
- [ ] Update RP Wiki Page with any changes or corrections.
- [ ] Once research lead has signed off, add it to the PM/Research lead agenda
- [ ] sign-off by PM 
- [ ] Does this issue have any dependency checkboxes in the action items here?
   - [ ] if yes, remove dependencies that are listed below (issues that are waiting on this issue to be completed.  Sometimes they have already been made and sometimes they yet to be made).
       - [ ] close this issue
   - [ ] if no, check to see if the followup issues have already been made (see related structure issue).  If new issues have not been made, then apply label `ready for research lead` and define what needs to be done in a comment below
      - [ ] close the issue
      - [ ] move to `Question/Review` Column


### Resources
- [Wiki Research Output Overview Page](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
- [Wiki [TYPE] Research Overview Page]
- [RP__ Wiki Page]
- [Google Drive Research by Type Folder](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=sharing)
- [Google Drive RP__ Folder]
- [Research Plan Template](https://docs.google.com/document/d/1SDW6n086PnSrrVSmPgQZWhlkz5_yC70vDvp7Lx7u0Wo/template/preview)
- [TWE: How to Write a Research Plan](https://docs.google.com/document/d/1Cwc0w4ZPUI8989w3jU8BW2LzLK_Tl5gHdI0VxN5ej0o/edit)

**Comment by ExperimentsInHonesty** [comment]
We have updated the template with a new versions.  The only changes made were formatting.
